LEAF_Request_Portal/admin/templates/view_disabled_fields.tpl

```diff
@@ -1,21 +1,24 @@
-<div class="leaf-center-content">
+<!--<div class="leaf-center-content">-->
 
+<div style="margin: 2em;">
```

Comment:
Nothing needs changed here at this time.  Just since I have your attention, keep in mind that hard values for style are going to be pulled out of templates with UI v2.  So keep this in the back of your mind.